public/stylesheets/scss/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$font-light: #B4C4CF;
```

Comment:
This is probably my fault, but could we label these by color instead of by the element they modify? `$blue-{light | lighter}`